node/tests/TestUtilities.ts

```diff
@@ -117,6 +126,33 @@ export function parseCommandLineArgs() {
     return parseArgs(process.argv.slice(2));
 }
 
+export async function testTeardown(
+    cluster_mode: boolean,
+    option: BaseClientConfiguration,
+) {
+    const client = cluster_mode
+        ? await RedisClusterClient.createClient(option)
+        : await RedisClient.createClient(option);
+
+    await client.customCommand(["FLUSHALL"]);
+    client.close();
+}
+
+export const getOptions = (
+    addresses: [string, number][],
+    protocol: ProtocolVersion,
```

Comment:
If protocol version and timeout is always the same, why do you need it as param and not as constant? 
If in the future you'll need it you can change, but its look like unnececry.

Reply:
its not always the same, its being used in other places as well